src/diffusers/pipelines/__init__.py

```diff
@@ -35,6 +36,14 @@
 else:
     from .audio_diffusion import AudioDiffusionPipeline, Mel
 
+try:
+    if not (is_torch_available() and is_note_seq_available()) and is_onnx_available()):
```

Comment:
```suggestion
    if not (is_torch_available() and is_note_seq_available())):
```

Sorry upon second reflection let's maybe not make it dependent on ONNX and instead allow to return raw mel-gan optionally. This would also make it much easier to test the model (see comments in pipeline)